lib/ui/widgets/menu_drawer.dart

```diff
@@ -84,13 +77,23 @@ class MenuDrawer extends StatelessWidget {
                         ),
                       ),
                     ),
+                    Positioned.fill(
+                      child: Material(
+                        color: Colors.transparent,
+                        child: InkWell(
+                          onTap: () {
+                            routerDelegate.push(TypedMaterialPage(
```

Comment:
Strange opening animation where fade in comes from below on andriod

Reply:
Resolving this on this PR, this will be looked at after PR (#157) has been merged